### PR TITLE
Fix various typos and spelling errors in documentation, comments and code.

### PR DESCRIPTION
**Pull request by thomasrenes**
Manually fixed some typos and spelling errors found with [typos](https://github.com/crate-ci/typos). This should improve the documentation and code quality.

Changes proposed in this pull request:

- Fix various typos and spelling errors in documentation, comments and code.

How to test the feature manually:

1. N/A

Pull request checklist:

- [X] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: cli/i18n/I18nData.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ public function __construct($data, $ignore) {
 		$this->data = $data;
 		$this->ignore = $ignore;
 
-		$this->synchonizeKeys();
+		$this->synchronizeKeys();
```
Comment (aledeg) [inline cli/i18n/I18nData.php]: This method has been removed

**Review by aledeg — CHANGES_REQUESTED** [review]
Thank you for your contribution. Nice work.

Could you rebase your branch to include the latest changes?

**Comment by thomasrenes** [comment]
> Thank you for your contribution. Nice work.
> 
> Could you rebase your branch to include the latest changes?

Thank you! I've rebased this branch.

**Review by Alkarex — APPROVED** [review]

**Review by aledeg — APPROVED** [review]

**Comment by Alkarex** [comment]
@thomasrenes Thanks, welcome, and please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md

We could consider adding *typos*  to our CI. See example of `shfmt`, which we are using, and which is also a go package. PR welcome anyone

https://github.com/FreshRSS/FreshRSS/blob/916df412f5b6f7fb9bcfb705a3c8c23e35304410/.github/workflows/tests.yml#L95-L97

**Comment by Alkarex** [comment]
Continues in https://github.com/FreshRSS/FreshRSS/pull/4138

**Pull request merged**